src/ConnectedMode/UI/Resources/Styles.xaml

```diff
@@ -87,4 +87,18 @@
         <Setter Property="BaselineAlignment" Value="TextBottom" />
     </Style>
 
+    <DataTemplate x:Key="NameAndKeyListBoxItem">
```

Comment:
Not sure if the `DataTemplate` belongs to the `Styles.xaml` it feels irrelevant, can it be extracted to another file?

Reply:
I think it's fine because this template is used for display (ie styling)

Reply:
Not exactly, because it contains details about the layout and the data. I am not sure if this will have an impact on the other xaml files that import `Styles.xaml`. @gabriela-trutan-sonarsource ?

Reply:
I think it is safe to add DataTemplate to Styles.xaml. It should not impact any other file, because it has a key defined. So as long as no control references that key, it will have no impact.